modular_skyrat/master_files/code/modules/clothing/under/akula_jobs.dm

```diff
@@ -0,0 +1,302 @@
+	icon_state = "sec"
+	base_icon_state = "sec"
+	armor_type = /datum/armor/rank_security
+
+/obj/item/clothing/under/akula_wetsuit/job/command
+	name = "Command Shoredress wetsuit"
+	desc = "The 'Command'-Type Shoredress is yet another model commissioned by NanoTrasen; but the origins of this wetsuit lie in designs belonging to, \
+			typically, high-ranking officials and managers in the Old Principalities. \n\
+			The bright luminescent panels on the arms have been further set apart by similar paneling on the chest, meant to ensure the wearer looks distinct both in the water, on land, and even on camera. \n\
+			The temperature systems have been upgraded, as well as the choice to use more comfortable fabrics in the construction.\n\n\ "
+	icon_state = "command"
+	base_icon_state = "command"
+
+
+/obj/item/clothing/head/helmet/space/akula_wetsuit
+	name = "Shoredress helm"
+	desc = "Known simply as a 'Glass' throughout Azulean society as a whole, these spheroidal helmets are often the main source of comfort for workers on land; domestic and abroad. \
+		More advanced than humans would ever give them credit for, a Shoredress's Glass is a piece of technology unto itself. \n\n\
+		These helmets employ a near-invisible system of cameras and sensors to prevent refraction from the water kept inside. \
+		The 'flexiglass' glass comprising the unit is chemically strengthened to be thin, light, and damage-resistant, but capable of bending even in half without shattering; all to allow you to touch your face. \n\
+		Some have taken to putting electronic displays around the face to help express emotion, or to signal nonverbally. \
+		These helms are normally attached to Shoredresses or Stardresses, but comes with a fitted neoprene collar to allow wear on essentially anything.\n\n\ "
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/head/akula.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head/akula.dmi'
+	clothing_flags = STOPSPRESSUREDAMAGE | THICKMATERIAL | SNUG_FIT | PLASMAMAN_HELMET_EXEMPT
+	icon_state = "helmet"
+	inhand_icon_state = "helmet"
+	strip_delay = 6 SECONDS
+	armor_type = /datum/armor/wetsuit_helmet
+	resistance_flags = FIRE_PROOF
+	/// Variable for storing hats which are worn inside the bubble helmet
+	var/obj/item/clothing/head/attached_hat
+	flags_inv = null
+	flags_cover = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF
+
+	/// Helmet armor
+/datum/armor/wetsuit_helmet
+	bio = 100
+	fire = 100
+
+/obj/item/clothing/head/helmet/space/akula_wetsuit/Initialize(mapload)
+	. = ..()
+	update_appearance()
+
+/obj/item/clothing/head/helmet/space/akula_wetsuit/Destroy()
+	. = ..()
+	var/mob/user = loc
+	if(attached_hat)
+		attached_hat.forceMove(drop_location())
+	if(!istype(user))
+		return
+	remove_wetsuit_status_effect(user)
+
+/obj/item/clothing/head/helmet/space/akula_wetsuit/equipped(mob/user, slot)
+	. = ..()
+	apply_wetsuit_status_effect(user)
```

Comment:
All of this copy-pasting wouldn't be necessary if you made that behavior into a component. Just poke me on Discord if you need help understanding how to do that, but I'm fairly sure you've already made components before(?)